assets/src/edit-story/utils/usePreventWindowUnload.js

```diff
@@ -50,7 +50,10 @@ function usePreventWindowUnload() {
     },
     [context]
   );
-
+  const isDevelopment = process.env.NODE_ENV === 'development';
+  if (isDevelopment) {
```

Comment:
What about creating a global constant `DEVELOPMENT_ENV = process.env.NODE_ENV === 'development'` and approach this like `useWhyDidYouUpdate`? 

E.g. `export default DEVELOPMENT_ENV ? () => () => {} : usePreventWindowUnload;`

It avoids run the code above unnecessarily.

Reply:
Brilliant, I wanted to do early return but in hooks - you can't, so I moved the code down and down.

About global constant: I'm fine with both (it's just 2 places now).